cclib/method/volume.py

```diff
@@ -136,9 +139,16 @@ def scinotation(num):
     return ("%sE%s%s" % (broken[0], sign, broken[1][-2:])).rjust(12)
 
 
-def getbfs(coords, gbasis):
-    """Convenience function for both wavefunction and density based on PyQuante Ints.py."""
-    mymol = cclib2pyquante.makepyquante(coords, [0 for _ in coords])
+def getbfs(ccmol):
+    """Convenience function for both wavefunction and density based on PyQuante Ints.py.
+    
+    Input:
+        ccmol -- ccData object

```

Comment:
Perhaps call this `data` or `ccdata`.

Reply:
This applies to PR #896 as well, so I've modified both of those. Thanks!